docs/changes.md

```diff
@@ -0,0 +1,12 @@
+# News and Updates
```

Comment:
Ah nice, I like "News and Updates" as the title for the page. Perhaps we can make that the page title in the mkdocs config as well, and then the path can just be `/news/`?

Reply:
oh, haha, I changed it from "News and Updates" to "Recent Changes" after reading your story but missed a spot here. Personally I was more sold on "Recent Changes" because it's hard to say whether any of this is actually "news". But we can definitely change back